app/src/main/java/com/gogiyumandroid/ui/home/HomeFragment.kt

```diff
@@ -1,31 +1,85 @@
+@file:Suppress("DEPRECATION")
+
 package com.gogiyumandroid.ui.home
 
 import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
-import android.widget.TextView
+import android.widget.ImageView
+import android.widget.TableLayout
+import android.widget.TableRow
 import androidx.fragment.app.Fragment
-import androidx.lifecycle.Observer
 import androidx.lifecycle.ViewModelProviders
+import com.bumptech.glide.Glide
 import com.gogiyumandroid.R
+import kotlinx.android.synthetic.main.custom_table_row.view.*
+import okhttp3.*
+import org.json.JSONArray
+import java.io.IOException
+
 
 class HomeFragment : Fragment() {
 
     private lateinit var homeViewModel: HomeViewModel
+    private val myGogiyumAddress = "https://gogiyum.com/food.json"
 
     override fun onCreateView(
-        inflater: LayoutInflater,
-        container: ViewGroup?,
-        savedInstanceState: Bundle?
+            inflater: LayoutInflater,
+            container: ViewGroup?,
+            savedInstanceState: Bundle?
     ): View? {
         homeViewModel =
-            ViewModelProviders.of(this).get(HomeViewModel::class.java)
+                ViewModelProviders.of(this).get(HomeViewModel::class.java)
+
+
+
         val root = inflater.inflate(R.layout.fragment_home, container, false)
-//        val textView: TextView = root.findViewById(R.id.text_home)
```

Comment:
Can you remove unused code? (commented code)

Reply:
I erased it